openapi-validator/src/main/java/io/ballerina/openapi/validator/Constants.java

```diff
@@ -53,16 +66,68 @@ public class Constants {
     public static final String RECORD = "record";
     public static final String NUMBER = "number";
     public static final String INT = "int";
-    public static final String USER_DIR = "user.dir";
-    public static final String FALSE = "false";
-    public static final String SLASH = "/";
-    public static final String FULL_STOP = ".";
-    public static final String HTTP_PAYLOAD = "http:Payload";
+    public static final String FLOAT = "float";
+    public static final String DEFAULT = "default";
+    public static final String DOUBLE = "double";
+    public static final String HEADER_NAME = "name";
+    public static final String WILD_CARD_CONTENT_KEY = "*/*";
+    public static final String WILD_CARD_SUMMARY = "Any type of entity body";
+    public static final String MEDIA_TYPE = "mediaType";
 
-    /**
-     * Enum Type for handle the validation error types.
-     */
-    public enum Type {
-        STRING, INT, RECORD, OBJECT, ARRAY, BOOLEAN, DECIMAL, ANYDATA, INTEGER, NUMBER
+    public static final Map<String, String> HTTP_CODES;
```

Comment:
Shouldn't we have a map of types `<int,String>` instead? IMO it will be better to treat the status code as the key since we'll can be sure on having unique keys in that way

Reply:
This is used for the Balerina to OpenAPI validator in return we received some specific description about return code instead of having code. example : status code 200 represent in ballerina return as `http:Ok` therefore we need to get that unique description as our key to get the correct status code for validation. [reference](https://github.com/ballerina-platform/module-ballerina-http/blob/7954bdc99fb76ed22675fa11f8b803697a576eee/ballerina/http_status_code_types.bal#L19)
in OpenAPI code(200) given as a string value, therefor Map<String, String> used to make convenience the validation comparison